bundles/org.openhab.binding.wlanthermo/src/main/resources/OH-INF/thing/channel-group-types-esp32.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wlanthermo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- System Group ESP32 -->
+	<channel-group-type id="cg_system_esp32">
```

Comment:
Since the group types are each meant for a specific thing type, why not define the group types in the thing type files instead?

Reply:
I've split this up to have a better overview on the different things, channel-groups and channels of the hardware versions.
Does it have any benefit of putting it in the same file?

Reply:
The only benefit is it is slightly better organized and thus requires less jumping around files to fully understand the thing xml configurations. But changing it is entirely up to you. Feel free to keep it as it is.